plugins/frontend/haproxy-sni-proxy/README.haproxy-sni-proxy.md

```diff
@@ -93,28 +93,29 @@ For more information, please refer to the [OpenShift Origin Cartridge Developer'
 ## SNI Proxy Requirements and Configuration
 
 The SNI proxy reads its configuration from the OpenShift node
-configuration files.
+configuration files:
 ```
 /etc/openshift/node.conf
 /etc/openshift/node-plugins.d/openshift-origin-frontend-haproxy-sni-proxy.conf
 ```
 
 The SNI proxy is configured for ports 2303 through 2308 by default.
-
 This list is configurable through the "PROXY_PORTS" parameter in
 `openshift-origin-frontend-haproxy-sni-proxy.conf`.  Caution should be
-used changing the list to ensure that there are no conflicts,
+used when changing the list to ensure that there are no conflicts,
 including with the gear port proxy.
 
 The proxy ports should have a firewall policy similar to ports 80 and
-443 (HTTP and HTTPS).  Typically, allowing contact.
+443 (HTTP and HTTPS) to allow connections.
 
-Depending on the configuration, the port proxy will bind to loopback
-(127.0.0.1) and the IP address of the public facing network interface
-(eg: eth0).  If an IP address cannot be determined, then the port
-proxy will bind to INADDR_ANY.
+By default, the port proxy will bind to loopback (127.0.0.1) and the IP
+address of the public-facing network interface specified by the
+"EXTERNAL_ETH_DEV" setting in `node.conf` (e.g., eth0) or inferred from
+the "PUBLIC_IP" setting in `node.conf`.  The public-facing address can
+be overridden using the "BIND_IP" setting in
+`openshift-origin-frontend-haproxy-sni-proxy.conf`.  If an IP address
+cannot be determined, then the port proxy will bind to INADDR_ANY.
```

Comment:
As a general comment, it seems as though the markdown style is different than [what is used for OpenShift docs](https://github.com/openshift/openshift-docs/blob/master/contributing_to_docs/doc_guidelines.adoc#quick-markup-reference), so I withheld comments there.

Just a few comments from me. Otherwise, looks good! :ok_hand: